platforms/ide/problems/src/main/kotlin/org/gradle/problems/internal/impl/DefaultProblemsReportCreator.kt

```diff
@@ -96,12 +96,13 @@ class DefaultProblemsReportCreator(
     }
 }
 
+@Suppress("USELESS_ELVIS")
```

Comment:
🤔 If it can be actually be nullable, should we mark the getter nullable?
I think the constructor should be still non-nullable, so people are warned that they shouldn't pass it.

Reply:
I think this would cause a lot of confusion, `NonNull` input but `Nullable` return value from the getter.
I consider this a temporary measure.